docs/lib/duckdb.md

```diff
@@ -124,42 +124,41 @@ LOAD h3;
 SELECT format('{:x}', h3_latlng_to_cell(37.77, -122.43, 9)) AS cell_id;
 ```
 
-Beyond the official extensions repositories (with core extensions at `https://extensions.duckdb.org` and community extensions at `https://community.duckdb.org`), you can install an extension from an explicit URL:
+Beyond the official extensions repositories (with `core` extensions at `https://extensions.duckdb.org` and `community` extensions at `https://community.duckdb.org`), you can install an extension from an explicit URL:
 
 ```sql echo run=false
-INSTALL custom FROM 'https://example.com/v1.1.1/wasm_mvp/custom.wasm';
-```
-
-### Self-hosted core extensions
-
-Framework downloads a copy of the [core extensions](https://duckdb.org/2023/12/18/duckdb-extensions-in-wasm.html), and the DuckDBClient installs them by default. This ensures that all the common extensions ("json", "inet", "spatial", etc.), are self-hosted.
-
-You can however override this (for example, if you need to test something against a new version of an extension), and install explicitly:
-
-```sql echo run=false
-INSTALL json FROM core;
--- use JSON features
+INSTALL custom FROM 'https://example.com/';
```

Comment:
I think we should discourage people from installing extensions from within SQL blocks: doing so globally changes the behavior of the DuckDBClient instance and can lead to race conditions/nondeterministic behavior across blocks, and also because we want to favor self-hosting of extensions rather than hotlinking to an external website.

The recommended way to install extensions should be via the front matter or the project config (or to do it in JavaScript by redefining the `sql` literal and awaiting the loading of the extensions).